stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -2,6 +2,8 @@
 
 from typing import Any, Callable, Iterable, Mapping
 
+from multiprocessing.process import current_process as current_process
```

Comment:
Hmm. It's there

```
$ cat stdlib/3/multiprocessing/process.pyi 
def current_process(): ...
def active_children(): ...
```

Am I mispelling it?  Or do I have to import it in a different way?